Allows construction of airtight plastic flaps

mikomyazaki opened the pull request:
Makes airtight flaps and normal flaps the same item, toggled by screwdriver.
Changes deconstruction to use crowbar instead.

Will allow people to set up new disposals chutes without causing atmos problems and to fix things if supply deconstructs their airtight flaps.

afterthought2 commented on maps/torch/torch-1.dmm:
```diff
@@ -1301,7 +1301,7 @@
 	id = "packageSort1"
 	},
-/obj/structure/plasticflaps/mining,
+/obj/structure/plasticflaps/,
```
Seems like it makes these mapped ones not airtight. Make an airtight subtype for mapping (just with the var set) and use here.

mikomyazaki replied:
Good point!

afterthought2 commented:
conflicts, and may as well squash while in there.

mikomyazaki commented:
Merge conflict resolved, squash is squish'd.

afterthought2 commented on maps/torch/torch2_deck4.dmm:
```diff
@@ -12026,7 +12026,7 @@
 	id = "packageSort1"
 	},
-/obj/structure/plasticflaps/mining,
+
```
Looks not correct.

Bxil commented on code/game/objects/structures/plasticflaps.dm:
```diff
@@ -41,11 +42,15 @@
+	if(isScrewdriver(W) && anchored)
+		airtight = !airtight
+		airtight ? become_airtight() : clear_airtight()
+		user.visible_message("<span class='warning'>\The [user] adjusts the \the [src], [airtight ? "preventing" : "allowing"] air flow.</span>")
```
Superfluous `the`.

mikomyazaki committed:
Fixes all issues from reviewers.

Makes airtight plastic flaps and plastic flaps the same item. Toggled with screwdriver.

Handles merge conflicts.

Resolving issues from PR review.

Did a derp.

Makes airtight plastic flaps and plastic flaps the same item. Toggled with screwdriver.

mikomyazaki commented:
Resolved issues.
